docs/source/conf.py

```diff
@@ -134,15 +134,27 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-import sphinx_rtd_theme
-html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+import alabaster
+html_theme_path = [alabaster.get_path()]
+extensions = ['alabaster']
```

Comment:
This line here is the problem. You're redefining the `extensions` variable which is already defined -- you should add `'alabaster'` in the list where it's already defined or append it rather than redefining the variable.